app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -10,174 +10,172 @@
+      android:layout_weight="1"
+      android:clickable="@{viewModel.completedStoryCount > 0}"
+      android:onClick="@{() -> viewModel.clickOnCompletedStoryCount()}"
+      app:cardElevation="4dp"
+      app:contentPadding="8dp"
       app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@+id/progress_stats_linear_layout">
+      app:layout_constraintHorizontal_weight="1"
+      app:layout_constraintStart_toEndOf="@+id/ongoing_topics_container"
+      app:layout_constraintTop_toBottomOf="@+id/profile_name_text_view">
 
-      <TextView
-        android:id="@+id/recently_played_stories_text_view"
-        android:layout_width="0dp"
+      <LinearLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="28dp"
-        android:layout_weight="1"
-        android:fontFamily="sans-serif-medium"
-        android:text="@string/recently_played_stories"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="18sp" />
-
-      <TextView
-        android:id="@+id/view_all_text_view"
-        android:layout_width="wrap_content"
-        android:layout_height="48dp"
-        android:layout_marginStart="8dp"
-        android:layout_marginEnd="28dp"
-        android:fontFamily="sans-serif-medium"
-        android:gravity="center_vertical"
-        android:onClick="@{() -> viewModel.clickOnViewAll()}"
-        android:text="@string/view_all"
-        android:textAllCaps="true"
-        android:textColor="@color/oppiaPrimaryDark"
-        android:textSize="14sp" />
-    </LinearLayout>
-  </LinearLayout>
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:orientation="vertical"
+        android:padding="8dp">
+
+        <TextView
+          android:id="@+id/completed_stories_count"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:fontFamily="sans-serif"
+          android:gravity="center"
+          android:text="@{Integer.toString(viewModel.completedStoryCount)}"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="24sp" />
+
+        <TextView
+          android:id="@+id/completed_stories_description_text_view"
+          android:layout_width="60dp"
+          android:layout_height="wrap_content"
+          android:fontFamily="sans-serif"
+          android:gravity="center"
+          android:text="@{viewModel.completedStoryCount > 1 ? @string/stories_completed : @string/story_completed}"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="12sp" />
+      </LinearLayout>
+    </com.google.android.material.card.MaterialCardView>
+
+    <TextView
+      android:id="@+id/recently_played_stories_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="48dp"
```

Comment:
Instead of giving height as 48dp. Give minheight as 48dp.

Reply:
Done.